application/app/controllers/dataverse/landing_page_controller.rb

```diff
@@ -6,7 +6,7 @@ def index
       hub_registry = DataverseHubRegistry.registry
       installations = hub_registry.installations
       page = params[:page] ? params[:page].to_i : 1
-      @installations_page = Page.new(installations, page, 25)
+      @installations_page = Page.new(installations, page, 25, q: params[:q], filter_by: :name)
```

Comment:
Why not Page.query? To make it more meaningful and consistent with other controllers?

Reply:
fixed